src/services/recoverPassword/RecoverPasswordService.tsx

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+import Endpoints from 'constants/endpoints';
+
+const RecoverPassowrdService = async (email: string) => {
```

Comment:
typo here and in the export:
```suggestion
const RecoverPasswordService = async (email: string) => {
```